### ISSUE TITLE
Switch

### ISSUE THREAD

**Issue by StreiffBerenice** [Orange-OpenSource/Orange-Boosted-Angular#64]
@liyokuna - Ok for the vocalisation of the switches's states
But 👍 ----
when the switches are checked, the colour contrast isn't sufficient for people who have a visual disability -> you should use black for checked switches.
[See switch component](https://deploy-preview-47--ng-boosted.netlify.com/#/documentation/o-switch)

**Comment by liyokuna** [comment]
@PigeardSylvain 
to be compliant with the design can we easily change the color ? In order to be more accessible.

**Comment by PigeardSylvain** [comment]
Changing the checbox tick to black is not a good solution :
![image](https://user-images.githubusercontent.com/13221271/59766513-a8725a80-92a0-11e9-97b8-bb5c597f013b.png)

An opinion on the subject @CyriaqueBillard ?

**Comment by CyriaqueBillard** [comment]
I do not have a final opinion on that as we need to consider all the components we have that are currently using orange as an active color. If we change anything for the toggle switch, then we have to change radio buttons and checkboxes accordingly. We can not decide only for one item as we need to stay consistent across our UI components system.
At the moment, my understanding is that in order to increase accessibility we should do our best to no always be relying on color but also on change of shape. The toggle switch we are discussing here has color change and shape change. Its "off" state is black (not gray) with a white cross. Its "On" state is orange with a white tick. This being said, I do understand that a white tick is less prominent than a black tick. Now, considering the change of shape I mentioned above, do we know in which proportion a black tick would increase the accessibility?
